commands/rm.go

```diff
@@ -72,6 +90,7 @@ func rmCmd(dockerCli command.Cli, rootOpts *rootOptions) *cobra.Command {
 	flags := cmd.Flags()
 	flags.BoolVar(&options.keepState, "keep-state", false, "Keep BuildKit state")
 	flags.BoolVar(&options.keepDaemon, "keep-daemon", false, "Keep the buildkitd daemon running")
+	flags.BoolVar(&options.inactive, "inactive", false, "Remove inactive builders")
```

Comment:
I prefer `all-inactive` that makes it clear that this is a different mode

Reply:
Agree but more generally I think the CLI has a confusing design for some commands. For example the following would make more sense:

* `docker buildx du` > `docker buildx cache du`
* `docker buildx prune` > `docker buildx cache prune`
* `docker buildx rm --all-inactive` > `docker buildx prune`

Reply:
Using `prune` for something other than build cache is quite confusing imho. Especially because it means completely different thing atm. For the possibility of cache subcommand we can discuss in another issue.

Reply:
> Using `prune` for something other than build cache is quite confusing imho.

Yes can be confusing.

> For the possibility of cache subcommand we can discuss in another issue.

:+1: